SPECS/hdf5/hdf5.spec

```
@@ -0,0 +1,1054 @@
+%global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
+# No more Java on i686
+%ifarch %{java_arches}
+%bcond_without java
+%else
+%bcond_with java
+%endif
+%global version_main %(echo %{version} | cut -d. -f-2)
```

I think this should fix the CGManifest check.

```suggestion
%define version_main %(echo %{version} | cut -d. -f-2)
```